feat: add docs (#192)
FuelLabs/fuels-wallet: README.md

## ⚡️ Fuel Wallet

The official wallet of the fastest modular execution layer: [Fuel](https://fuel.network).

> FuelWallet is a crypto wallet, used to interact with Fuel Network. This project, enable users to access their assets and interacting with decentralized application, through a browser extension.

Reviewer: ```suggestion
FuelWallet is a crypto wallet used to interact with the Fuel Network. This project enables users to access their assets and interact with decentralized applications through a browser extension.
```

[![Install Wallet](docs/assets/install-button.png)](./docs/INSTALL.md)

Integrate your **DApp** using the [Wallet SDK](./docs/WALLET_SDK.md)

## 📗 Table of contents

- [📦 Install Wallet](./docs/INSTALL.md)
- [🧰 Fuel Wallet SDK](./docs/WALLET_SDK.md)
  - [👨‍💻 - Quickstart](./docs/WALLET_SDK.md#quickstart)
  - [🔗 - Request connection](./docs/WALLET_SDK.md#request-connection)
  - [📗 - List user accounts](./docs/WALLET_SDK.md#list-user-accounts)
  - [✍️ - Signing a message](./docs/WALLET_SDK.md#signing-a-message)
- [Contributing](./docs/GETTING_STARTED.md)
  - [Requirements](./docs/GETTING_STARTED.md#requirements)
  - [Running Project Locally](./docs/GETTING_STARTED.md#running-project-locally)

>     - [Getting the Repository](./docs/GETTING_STARTED.md#---getting-the-repository)

Reviewer: ```suggestion
    - [Getting the repository](./docs/GETTING_STARTED.md#---getting-the-repository)
```

>     - [Install Dependencies](./docs/GETTING_STARTED.md#---install-dependencies)

Reviewer: ```suggestion
    - [Install dependencies](./docs/GETTING_STARTED.md#---install-dependencies)
```

>     - [Run Local Node](./docs/GETTING_STARTED.md#---run-local-node)

Reviewer: ```suggestion
    - [Run a local node](./docs/GETTING_STARTED.md#---run-local-node)
```

>     - [Run Web App](./docs/GETTING_STARTED.md#---run-web-app)

Reviewer: ```suggestion
    - [Run the web app](./docs/GETTING_STARTED.md#---run-web-app)
```

>   - [Project Overview](./docs/GETTING_STARTED.md#-project-overview)

Reviewer: ```suggestion
  - [Project overview](./docs/GETTING_STARTED.md#-project-overview)
```

>   - [Useful Scripts](./docs/GETTING_STARTED.md#-useful-scripts)

Reviewer: ```suggestion
  - [Useful scripts](./docs/GETTING_STARTED.md#-useful-scripts)
```

  - [Running Tests](./docs/GETTING_STARTED.md#running-tests)
    - [Run Tests in Development Mode](./docs/GETTING_STARTED.md#run-tests-in-development-mode)
    - [Run Tests on a Local Test Environment](./docs/GETTING_STARTED.md#run-tests-on-a-local-test-environment)
- [Contribution Guide](./docs/CONTRIBUTING.md)
  - [Finding Something to Work On](./docs/CONTRIBUTING.md#finding-something-to-work-on)
  - [Contribution Flow](./docs/CONTRIBUTING.md#contribution-flow)

> | This project is under active development. Do not treat as production ready Wallet.

Reviewer: ```suggestion
| This project is under active development. Please do not treat it as a production-ready wallet.
```
